audit_test.go

```diff
@@ -748,6 +749,39 @@ func TestAuditClientGetStatusAsync(t *testing.T) {
 	}
 }
 
+func TestAuditClientSetImmutable(t *testing.T) {
+	if os.Geteuid() != 0 {
+		t.Skip("must be root to enable audit")
+	}
+	if !*immutable {
+		t.Skip("use -immutable to test SetImmutable")
+	}
+
+	var dumper io.WriteCloser
+	if *hexdump {
+		dumper = hex.Dumper(os.Stdout)
+		defer dumper.Close()
+	}
+
+	c, err := NewAuditClient(dumper)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer c.Close()
+
+	err = c.SetImmutable(WaitForReply)
```

Comment:
is this going to cause problems with CI infrastructure? i'm assuming we don't re-use boxes between runs so we can modify kernel state?

Reply:
nvm, looks like we're using travis for this library, should be good